### DISCUSSION TITLE
How to gotty https session

### DISCUSSION THREAD

**Discussion by gauravpareekk** [sorenisanerd/gotty#73] [Q&A]
Hi,

I'm building a web app where users can deploy apps to servers. If i've to share server terminal using gotty. 

I'm running gotty on 10050. But the issue is if I take that url and open outside my web app they can still access the terminal.

How can I restrict gotty access just from particular IP or server?

**Comment by booyaa** [comment]
You can't you need to add a WAF like CloudFlare to block or allow a specific IP address.
